action.yml

```diff
@@ -5,17 +5,52 @@ runs:
   using: composite
   steps:
 
+    - name: Install PHP
+      uses: shivammathur/setup-php@1a18b2267f80291a81ca1d33e7c851fe09e7dfc4 # v2.22.0
+      with:
+        php-version: 8.1
+
     - name: Generate tag name
       id: generate_tag_name
       shell: bash
       env:
         GITHUB_REF: ${{ github.ref }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         # refs/tags/0.1.23 => 0.1.23
         TAG=$(echo $GITHUB_REF | cut -c 11-)
         if ! [[ $TAG =~ ^([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
-          echo "Invalid semver tag $TAG"
-          exit 1
+          echo "Invalid semver tag $TAG, will use highest semver tag on repo instead"
```

Comment:
We should check what the context (`github.event_name`) is for this run. If this is a tag event and the tag isn't semver, we should just exit out instead of using the highest semver tag (which would have had this workflow run on in independently).

Reply:
Updated